hedera-node/hedera-app/src/main/java/com/hedera/node/app/meta/HandleScope.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (C) 2023 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hedera.node.app.meta;
+
+import static java.lang.annotation.RetentionPolicy.RUNTIME;
+
+import java.lang.annotation.Retention;
+import javax.inject.Scope;
+
+/**
+ * A Dagger scope that indicates that a component's lifetime is for each handle.
+ */
+@Scope
+@Retention(RUNTIME)
+public @interface HandleScope {}
```

Comment:
hmmm.  Is the scope properly to Handle (kind of implies each method), or each transaction?  Would Transaction (i.e. grpc `@CallScope`) be clearer here?

Reply:
Yeah it per each `handle` call which happens for each transaction. @CallScope, might be ambiguous on which call